Dockerfile

```diff
@@ -5,10 +5,9 @@ RUN mv composer.phar /usr/local/bin/composer
 RUN chmod a+x /usr/local/bin/composer
 RUN apt-get update && apt-get  install -y git unzip
 ARG VERSION
-RUN if [ "$VERSION" = "7.2" ]; then \
-    composer global require phpunit/phpunit:^7 && pecl install xdebug-2.7.0alpha1; else \
-    composer global require phpunit/phpunit:^4 && pecl install xdebug-2.5.5; fi
-RUN docker-php-ext-enable xdebug
 RUN composer global require hirak/prestissimo
+RUN composer global require phpunit/phpunit:^7
```

Comment:
@fkupper you need to specify a compatible PHPUnit version for php `7.1`

Reply:
Thanks for pointing out. I am a bit busy these last few days.

Will fix this in the weekend.

Reply:
@en0ma  This is funny because phpunit `7` should be compatible with php `7.1` to `7.3` .

https://phpunit.de/supported-versions.html
https://packagist.org/packages/phpunit/phpunit#7.5.14